etc/production.bash

```diff
@@ -138,6 +138,14 @@ tar xf libs.tar
 cp ${etc}/emotiq.bash.${kind} ${production_dir}/emotiq.bash
 cd ${lib}
 rm *
-# < /test >
 
-# cp ${???}/*.so* ${production_dir}
+cd ${production_dir}
+case ${kind} in
+    linux) 
+	tar cfj ${kind}.emotiq.bz2 em* *.so*
```

Comment:
Please use an explicit manifest for archiving items rather than glob patterns for artifacts present on the filesystem.   

Otherwise, the distribution process will possibly pick up stray files.

Reply:
I can name each of the files explicitly.  Is there something better?

Reply:
The better way would be to follow DRY (Do not repeat yourself) whereby you record the contents which you move into the production directory, and use this to drive the process.